pandas/io/formats/style.py

```diff
@@ -143,11 +143,16 @@ class Styler(StylerRenderer):
         uses ``pandas.options.styler.format.formatter``.
 
         .. versionadded:: 1.4.0
+    css : dict, optional
+        A dict of strings used to replace the default CSS classes described below.
```

Comment:
needs versionadded

Reply:
this PR follows #43686, so effectively splits this up into manageable chunks. Can you review that one first.